src/main.py

```diff
@@ -1,14 +1,21 @@
-import parse_fasta
-import sam_to_fasta_file
+from pathlib import Path
+
+from src import parse_fasta
+from src import sam_to_fasta_file
 
 # this method starts the tasks for the whole exercise (1.4 and 2.2 to be precise)
 
 if __name__ == '__main__':
     print(parse_fasta.map_reads(
-        r"/Users/Aline/Desktop/21HS/21HS PrBi/PrBi Alexander Kanitz/PrBi/RNA-Seq/Test files/sequences.fasta",
-        r"/Users/Aline/Desktop/21HS/21HS PrBi/PrBi Alexander Kanitz/PrBi/RNA-Seq/Test files/genome.fasta"))
-    sam_to_fasta_file.sam_to_fasta_file('Aligned.out.sam')
-    dict = sam_to_fasta_file.parse_fasta.map_reads(
```

Comment:
That was a bit of a weird call: it works because you import `parse_fasta` in `sam_to_fasta_file`, but you can of course also import `parse_fasta` directly. Or better yet, just import the functions you really need from the modules with e.g.,:

```
from parse_fasta import map_reads
```